lib/service.js

```diff
@@ -143,13 +142,11 @@
          * @example
          *
          *      // List all properties in the 'props.conf' file
-         *      var files = svc.configurations();
-         *      files.item("props", function(err, propsFile) {
-         *          propsFile.fetch(function(err, props) {
-         *              console.log(props.properties()); 
-         *          });
-         *      });
-         *
+         *      let files = svc.configurations();
+         *      let propsFile = await files.item();
```

Comment:
I think this should be `let propsFile = await files.item("props");` does that seem right?

Reply:
yes, and we have updated it